kong/pdk/response.lua

```diff
@@ -861,21 +890,23 @@ local function new(self, major_version)
         error("body must be a nil or a string", 2)
       end
 
-      if body then
-        if status < 400 then
-          -- only sends body to the client for 200 status code
-          local res, err = ngx.print(body)
-          if not res then
-            error("unable to send body to client: " .. err, 2)
-          end
+      local ctx = ngx.ctx
+      ctx.KONG_EXITED = true
 
-        else
-          self.log.err("unable to proxy stream connection, " ..
-                       "status: " .. status .. ", err: ", body)
-        end
-      end
+      if ctx.delay_response and not ctx.delayed_response then
+        ngx.log(ngx.ERR, "STREAM DELAYED")
```

Comment:
is this a real error, or a debug leftover?

Reply:
Yes, a left-over, thank you!